src/dxvk/dxvk_cmdlist.cpp

```diff
@@ -206,4 +206,15 @@ namespace dxvk {
     return m_vkd->vkQueueSubmit(queue, 1, &submitInfo, fence);
   }
   
-}
\ No newline at end of file
+  void DxvkCommandList::cmdBeginDebugUtilsLabel(VkDebugUtilsLabelEXT *pLabelInfo) {
+    m_device->instance()->vki()->vkCmdBeginDebugUtilsLabelEXT(m_execBuffer, pLabelInfo);
```

Comment:
we could also store the instance dispatch table in the command list somewhere to make this a bit less silly

Reply:
Sure.